### PR TITLE
Documentation for custom Node Selectors

### PR DESCRIPTION
**Pull request by jvc5546**

### CODE REVIEW

File: src/content/self-hosted/helm-configuration.mdx [hunk 1]
```diff
@@ -493,6 +493,22 @@ globals:
       effect: "NoSchedule"
 ```
 
+- `nodeSelectors`: Node selectors to be added to Pods that are either under the `okteto` or `dev` categories, as shown below. This helps you match Pods to nodes whose labels match the node selecotr.
```
Comment (pchico83) [inline src/content/self-hosted/helm-configuration.mdx:496]: should we document buildkit node selectors too?

File: src/content/self-hosted/helm-configuration.mdx [hunk 2]
```diff
@@ -493,6 +493,22 @@ globals:
...
+  - `okteto`: A dictionary of node selectors to be added to Pods related to Okteto components (i.e. Okteto api), which run in the `okteto` namespace. All the node selectors specified here will be applied in conjunction with the single node selector created by [`tolerations.oktetoPool`](self-hosted/helm-configuration.mdx#tolerations) until deprecation of `tolerations.oktetoPool`.
+  - `dev`: A dictionary of node selectors to be added to Pods created by user applications, which run in namespaces managed by Okteto. All the node selectors specified here will be applied in conjunction with the single node selector created by [`tolerations.devPool`](self-hosted/helm-configuration.mdx#tolerations) until deprecation of `tolerations.devPool`.
+
+```yaml
+globals:
+  nodeSelectors:
+    okteto:
+      oktetoSeletorOne: valueOne
```
Comment (pchico83) [inline src/content/self-hosted/helm-configuration.mdx:504]: Could we use a more realistic example? For example, to select nodes with `okteto-node-label: okteto` and ``okteto-node-label: dev`

**Review by pchico83 — APPROVED** [review]

**Comment by pchico83** [comment]
@jvc5546 merge from master to fix the build

File: src/content/self-hosted/helm-configuration.mdx (hunk 1 above)
Comment (jvc5546) [reply src/content/self-hosted/helm-configuration.mdx:497]: Good call, I forgot about those node selectors. They are documented now, thank you!

**Review by pchico83 — APPROVED** [review]
LGTM
@jvc5546 let's not merge until chart 1.20 is released and we switch the versions to do 1.21 the "unreleased" version. This feature didn't make to 1.20

**Pull request merged**